ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/TableEntityToolbar.tsx

```diff
@@ -18,6 +19,34 @@ type TableEntityToolbarProps = {
     children?: React.ReactNode;
 };
 
+const searchOptions: SearchOption[] = [
+    {
+        label: 'CVE',
+        value: 'CVE',
+        category: 'IMAGE_VULNERABILITIES',
+    },
+    {
+        label: 'Image',
+        value: 'IMAGE',
+        category: 'IMAGES',
+    },
+    {
+        label: 'Deployment',
+        value: 'DEPLOYMENT',
+        category: 'DEPLOYMENTS',
+    },
+    {
+        label: 'Namespace',
+        value: 'NAMESPACE',
+        category: 'NAMESPACES',
+    },
+    {
+        label: 'Cluster',
+        value: 'CLUSTER',
+        category: 'CLUSTERS',
+    },
+];
+
```

Comment:
What do you think about having the search options imported from a central location, like `searchUtils.tsx` or the search component itself? Just for the sake of avoiding string typos and repetition?

Something like
```ts
const searchOptions: SearchOption[] = [
    IMAGE_CVE_SEARCH_OPTION,
    IMAGE_SEARCH_OPTION,
    DEPLOYMENT_SEARCH_OPTION,
    NAMESPACE_SEARCH_OPTION,
    CLUSTER_SEARCH_OPTION,
];
```